base/cvd/cuttlefish/host/commands/cvd/server_command/host_tool_target.cpp

```diff
@@ -136,10 +141,14 @@ Result<FlagInfo> HostToolTarget::GetFlagInfo(
 
 Result<std::string> HostToolTarget::GetBinName(
     const std::string& operation) const {
-  CF_EXPECT(Contains(op_to_impl_map_, operation),
-            "Operation \"" << operation << "\" is not supported by "
-                           << "the host tool target object at "
-                           << artifacts_path_);
+  auto binary_find = OpToBinsMap().find(operation);
+  auto binaries = binary_find != OpToBinsMap().end()
+                      ? android::base::Join(binary_find->second, ", ")
+                      : "";
+  CF_EXPECTF(Contains(op_to_impl_map_, operation),
+             "Operation \"{}\" is not supported by the host tool target object "
+             "at \"{}\".  Looked for \"{}\" binaries to exist in \"{}/bin/\"",
+             operation, artifacts_path_, binaries, artifacts_path_);
```

Comment:
nit: either `Looked for \"{}\" binaries in` or `Expected \"{}\" binaries to exist in`

Reply:
Done